ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -34,7 +34,7 @@ database_user_group: postgres
 # This is the fully qualified domain name of your production server. Because
 # nginx checks this value against the URL being requested, it must be the same
 # as the server's DNS name. 
-site_fqdn: "97.107.132.199"
+site_fqdn: "104.197.237.109"
```

Comment:
Yes, the code comment notes need to be updated. In this case I think it needs to say, this can be a fqdn if you have DNS configured for your server, or it can also be an IP address.